axelrod/_strategy_utils.py

```diff
@@ -45,11 +45,19 @@ def limited_simulate_play(player_1, player_2, h1):
     player_2's strategy method to set any internal variables as needed. If you
     need a more complete simulation, see `simulate_play` in player.py. This
     function is specifically designed for the needs of MindReader."""
-    h2 = player_2.strategy(player_1)
+    h2 = cheating_bastard(player_1, player_2)
     update_history(player_1, h1)
     update_history(player_2, h2)
 
 
+def cheating_bastard(cheater, opponent):
```

Comment:
> I noticed one of your commit messages about mypy, any problems running that locally? (It's pip installable).

i had some weird issues that took working out.  for some reason, the mypy command doesn't make doesn't add to any kind of PATH variable.  so, with all this installed in a virtualenv i had to write a shell script pointing directly to a batch file the environment's Scripts folder.  all in all, the workaround needed 

- a python script to get names from run_mypy.py into a csv file
- the csv file
- a shell script to run the first one, load the second one and then run each variable through a path to the mypy.bat file.  

not pretty, but i've barely worked with bash before, so that was useful.